docker-compose.yml

```diff
@@ -44,7 +44,7 @@ services:
       - elasticsearch
   mongo:
     image: mongo:4.4
-    restart: always
+    restart: "no"
```

Comment:
Default is "no", so you can just remove all restart: "no" lines.

Reply:
Done.